include/votca/tools/eigen.h

```diff
@@ -27,6 +27,11 @@
 #define EIGEN_USE_MKL_ALL
 #endif
 
+#ifndef NOFFTW
+#include <fftw3.h>
```

Comment:
I don't think this line is needed as `/usr/include/eigen3/unsupported/Eigen/FFT` includes `fftw3.h` already.

Reply:
ahh good point.